docs/DeveloperGuide.md

```diff
@@ -257,12 +258,13 @@ _{Explain here how the data archiving feature will be implemented}_
 
 **Target user profile**:
 * Gamer who plays competitive games
-* Student in a University
+* Computer Science university student
 * Busy (can only play at specific times based on a weekly schedule)
 * Plays multiple online competitive multiplayer games (e.g., Apex legends, Minecraft, Valorant, etc)
-* Has a group of similar persona gamer friends who like him are busy
+* Has a group of similar persona gamer friends who like him are busy with their computer science university degrees
```

Comment:
maybe we can include a definition in the appendix/glossary for what 'busy' means

Reply:
we probably should